tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -664,9 +664,6 @@ public function checkItemValues($contribution) {
       $this->assertEquals(150.00, $dao->total_amount, 'Mismatch of total amount paid.');
       $this->assertEquals($dao->amount, array_pop($amounts), 'Mismatch of amount proportionally assigned to financial item');
     }
-
-    Contact::delete($this->_contactId);
-    Event::delete($this->_eventId);
```

Comment:
OK  - I've gone through that class much more thoroughly and switched to using a generic cleanup on it